### ISSUE TITLE
`knex-umzug` is not compatible with `umzug@^3.0.0`

### ISSUE THREAD

**Issue by leonardoraele** [marcbachmann/knex-umzug#25]
When trying to install `knex-umzug` on a project using `umzug` 3.0 or above, I get this error:

```npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: shared-notes-api@0.1.0
npm ERR! Found: umzug@3.0.0
npm ERR! node_modules/umzug
npm ERR!   umzug@"^3.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer umzug@"^2.3.0" from knex-umzug@4.0.0
npm ERR! node_modules/knex-umzug
npm ERR!   knex-umzug@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/leonardoraele/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/leonardoraele/.npm/_logs/2022-03-10T13_22_17_609Z-debug-0.log
```

**Comment by marcbachmann** [comment]
https://github.com/marcbachmann/knex-umzug/pull/26

**Comment by marcbachmann** [comment]
Published as version 4.1
